components/cam/src/physics/cam/scream_abortutils.F90

```diff
@@ -0,0 +1,51 @@
+module scream_abortutils
+
+!-------------------------------------------------
+!Utilities to stop the model in case of
+!catastrophic errors
+!-------------------------------------------------
+implicit none
+private
+
+!public subroutines
+public :: endscreamrun
+
+contains
+
+  subroutine endscreamrun (msg)
+    !-------------------------------------------------
+    ! This subroutine will print the optional message
+    ! received via optional arg "msg" and stop
+    ! the simulation
+    !-------------------------------------------------
+
+#ifdef SPMD
+    !Modules to use when SCREAM run with MPI under E3SM
+    use micro_p3_utils, only:iulog_e3sm
```

Comment:
It seems like this is never used. Can we remove it?

Reply:
I was initially implementing something like you mentioned above (using `#include mpif.h` etc.). In that case, I would need the above to put the error messages in the right log file. I may need it now since I am going to try what your suggested above.